framework/src/org/checkerframework/framework/qual/QualifierArgument.java

```diff
@@ -0,0 +1,24 @@
+package org.checkerframework.framework.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * An annotation to use on an element of a contract annotation to specify that the element specifies
```

Comment:
Please define "contract annotation".
The text is hard to read, such as "specify that the element specifies".  There are a lot of different parts and it's hard for a reader to keep them clear.

Please also add an example of use.  That will make the intention clearer.